react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,40 @@
+import { isArray } from './utils';
+
+export function min(...numbers) {
+  if (numbers.length === 0) {
+    return undefined;
+  }
+
+  if (numbers.length === 1) {
+    if (isArray(numbers[0])) {
+      return Math.min(...numbers[0]);
+    }
+
+    return numbers[0];
+  }
+
+  return Math.min(...numbers);
+}
+
+export function copy(myObject) {
+  if (isArray(myObject)) {
+    return myObject.slice();
+  }
+
+  return { ...myObject };
```

Comment:
this function can be a single line! like this `return bla bla bla`. Hint: use a ternary operator